Sources/Sentry/SentrySDK.m

```diff
@@ -383,6 +383,17 @@ + (void)installIntegrations
     }
 }
 
+/**
+ * Indicates that your screen is fully loaded and ready to be used.
+ * This should be used together with automatic performance transaction,
+ * a full display span will be created if there is a view controller transaction
+ * happening.
+ */
```

Comment:
`l`: We don't need comments in the `.m` file, as we already have them in the `.h` file.
```suggestion
```

Reply:
ctrl+c, ctrl+v 😁